src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -187,6 +196,28 @@ void deleteUser(String uid) throws FirebaseAuthException {
     }
   }
 
+  DownloadAccountResponse listUsers(int maxResults, String pageToken) throws FirebaseAuthException {
+    ImmutableMap.Builder<String, Object> builder = ImmutableMap.<String, Object>builder()
+        .put("maxResults", maxResults);
+    if (pageToken != null) {
+      checkArgument(!pageToken.isEmpty(), "invalid end of list page token");
```

Comment:
How about pageToken.equals(ListUsersPage.END_OF_LIST)? That way,  that implementation detail only exists in one place.

Reply:
Good call. Done.